components/buttons.tsx

```diff
@@ -28,26 +29,29 @@ export const LoadingButton = ({
 }: ButtonProps & {
   loading?: boolean
   spinnerProps?: Partial<Omit<SpinnerProps, "as" | "role" | "aria-hidden">>
-}) => (
-  <Button {...restProps} disabled={loading || disabled}>
-    {loading ? (
-      <>
-        <Spinner
-          as="span"
-          animation="border"
-          size="sm"
-          role="status"
-          aria-hidden
-          className="me-2"
-          {...spinnerProps}
-        />
-        <span className="visually-hidden">Loading...</span>
-      </>
-    ) : null}
+}) => {
+  const { t } = useTranslation("common")
+  return (
+    <Button {...restProps} disabled={loading || disabled}>
+      {loading ? (
+        <>
+          <Spinner
+            as="span"
+            animation="border"
+            size="sm"
+            role="status"
+            aria-hidden
+            className="me-2"
+            {...spinnerProps}
+          />
+          <span className="visually-hidden">{t("loading.inProgress")}</span>
```

Comment:
Couldn't manually verify this translation in dev, this span appears to be hidden. Should this be removed?